lms/services/file.py

```diff
@@ -21,6 +21,31 @@ def get(self, lms_id, type_):
             .one_or_none()
         )
 
+    def get_with_guid(self, lms_id, type_):
+        """
+        Return with the given lms_id and type_ that belongs to the same GUID as the current ApplicationInstance.
```

Comment:
We could argue that the real fix for this is then doing "Organization" or, before that, re-arrange the File table to have the "guid" column itself and probably having another `file_application_instance` n to m table.

I think those are valid points but given that organization are arriving soon. I rather get the bugfix out and deal with rearrange the DB in the context of the new organizations work.